### PR TITLE
Feature/stij 147

### PR DESCRIPTION
**Pull request by meirege**

### CODE REVIEW

File: components/21-atoms/inputs/input-multiselect-chosen/input-multiselect-chosen.binding.js [hunk 1]
```diff
@@ -15,7 +15,11 @@
    * Replace the mobile breadcrumb of Gent base by our own.
    */
   function initMultiselectChosen() {
-    $('.chosen-select').chosen({disable_search: true});
+    var isIpad = navigator.userAgent.match(/iPad/i) != null;
```
Comment (Jeroen005) [inline components/21-atoms/inputs/input-multiselect-chosen/input-multiselect-chosen.binding.js]: Does it work on iPhone?

Comment (meirege) [reply components/21-atoms/inputs/input-multiselect-chosen/input-multiselect-chosen.binding.js]: By default Chosen is disabled on mobile devices, yes. It's a chosen thing.

File: components/31-molecules/fields/multiselect-chosen/_multiselect-chosen.scss [hunk 2]
```diff
@@ -6,3 +6,11 @@
     line-height: 0;
   }
 }
+
+// sass-lint:disable no-important
```
Comment (Jeroen005) [inline components/31-molecules/fields/multiselect-chosen/_multiselect-chosen.scss]: We should disable it only on line 12. Now it's disabled for the entire file. https://github.com/sasstools/sass-lint#disabling-linters-via-source

Comment (meirege) [reply components/31-molecules/fields/multiselect-chosen/_multiselect-chosen.scss]: Fixed.

**Pull request merged**